elixir/apps/web/lib/web/live/policies/components.ex

```diff
@@ -1,9 +1,474 @@
+        </p>
+      </div>
+
+      <div
+        id="policy_conditions_provider_id_condition"
+        class="grid gap-2 sm:grid-cols-5 sm:gap-4 hidden"
+      >
+        <.input
+          type="select"
+          name="policy[conditions][provider_id][operator]"
+          id="policy_conditions_provider_id_operator"
+          field={condition_form[:operator]}
+          placeholder="Operator"
+          options={condition_operator_options(:provider_id)}
+          value={condition_form && condition_form[:operator].value}
+        />
+
+        <%= for {value, index} <- Enum.with_index((condition_form[:values] && condition_form[:values].value || []) ++ [nil]) do %>
+          <div :if={index > 0} class="text-right mt-3 text-sm text-neutral-900">
+            or
+          </div>
+
+          <div class="col-span-4">
+            <.input
+              type="select"
+              field={condition_form[:values]}
+              name="policy[conditions][provider_id][values][]"
+              id={"policy_conditions_provider_id_values_#{index}"}
+              options={[{"Select Provider", nil}] ++ Enum.map(@providers, &{&1.name, &1.id})}
+              value_index={index}
+              value={value}
+            />
+          </div>
+        <% end %>
+      </div>
+    </fieldset>
+    """
+  end
+
+  defp current_utc_datetime_condition_form(assigns) do
+    assigns = assign_new(assigns, :days_of_week, fn -> @days_of_week end)
+
+    ~H"""
+    <fieldset>
+      <% condition_form = find_condition_form(@form[:conditions], :current_utc_datetime) %>
+
+      <.input
+        type="hidden"
+        field={condition_form[:property]}
+        name="policy[conditions][current_utc_datetime][property]"
+        id="policy_conditions_current_utc_datetime_property"
+        value="current_utc_datetime"
+      />
+
+      <.input
+        type="hidden"
+        name="policy[conditions][current_utc_datetime][operator]"
+        id="policy_conditions_current_utc_datetime_operator"
+        field={condition_form[:operator]}
+        placeholder="Operator"
+        value={:is_in_day_of_week_time_ranges}
+      />
+
+      <div
+        class="cursor-pointer"
+        phx-click={
+          JS.toggle_class("hidden",
+            to: "#policy_conditions_current_utc_datetime_condition"
+          )
+          |> JS.toggle_class("hero-chevron-down",
+            to: "#policy_conditions_current_utc_datetime_chevron"
+          )
+          |> JS.toggle_class("hero-chevron-up",
+            to: "#policy_conditions_current_utc_datetime_chevron"
+          )
+        }
+      >
+        <legend>
+          <.icon
+            id="policy_conditions_current_utc_datetime_chevron"
+            name="hero-chevron-down"
+            class="w-5 h-5"
+          /> Current time
+        </legend>
+
+        <p class="text-sm text-neutral-500 mb-2">
+          Restrict access based on the current time of the day in UTC timezone.
```

Comment:
UX suggestion here. I know it's a bit more work, but what do you think of allowing them to select the timezone from a dropdown, defaulting to their browser's local timezone? It could just be a frontend change and we normalize to UTC before validation or something.

Otherwise the admin has to go and calculate the difference before this is usable.

Reply:
As I remember we were discussing this on standup and decided not to do this on the first iteration? But I'm all up for re-calculating timezones if you feel we should do it right away.

Reply:
Hm yeah I may have misunderstood. Do we need to recalculate though? I was thinking it would just be DateTime.new/3 with `Intl.DateTimeFormat().resolvedOptions().timeZone` from JS.

Reply:
@jamilbk this is not trivial. We can make a custom component and show datetimes in local format easily but mapping the time values back and forth is not trivial. The user inputs values in the selected (local by default timezone), then those values should be saved in UTC but still rendered in the input in the local timezone (otherwise numbers will be changed as you type).

Reply:
Also, shifting is not trivial. For example: `20:00-23:00` on Monday in UTC becomes `22:00-23:59` on Monday and `00:00-02:00` on Tuesday out of sudden. I think the way to handle it is to start storing the timezone in which the data was entered and re-calculate at evaluation time.